### DISCUSSION TITLE
Feature Request - Web page

### DISCUSSION THREAD

**Discussion by DeFlanko** [xnorpx/blue-onyx#39] [Ideas]
I like the stats page -- this is helpful for sure.  I do have some additional question about the statistics on this page.. what values are considered good? how can i use these metrics to indicate if my system is under performing? 

Would like to see if it's possible to have the stats refresh on their own.  As well as a link to the testing url (and while on the Testing url a link back to stats) 

it would be cool to see a real time log display, minus all the BASE64 processing.

**Comment by xnorpx** [comment]
Description of stats I think is fair, let me add discussion about it and write a little about the metrics.

Regarding the feature set of the webpages I simply did the minimal cleanest thing I could that I assumed people would like. I am not a web developer and I am neither good at it or enjoy it :)

But I am happy to take PR's with improvements to the webpages as long as they don't become an extra maintainer project by themself.  Like keep it simple but useful. Building the project should be fairly straight forward.

1 Install rust (and Windows build tool chains)
2 Checkout code
3 Open a visual studio developer console x64
4 cd to directory
5 cargo buiild
6 go grab a coffee for 5-20 minutes due to slow ONNX build.

**Comment by xnorpx** [comment]
Added stats discussion here:

https://github.com/xnorpx/blue-onyx/discussions/40

**Comment by xnorpx** [comment]
The new stats webpage automatically refreshes every 10 seconds. I added links from home to home.

**Comment by DeFlanko** [comment]
Love the webpage upgrades! closing this.
